pandas/tests/groupby/transform/test_numba.py

```diff
@@ -127,3 +127,19 @@ def func_1(values, index):
     with option_context("compute.use_numba", True):
         result = grouped.transform(func_1, engine=None)
     tm.assert_frame_equal(expected, result)
+
+
+@td.skip_if_no("numba", "0.46.0")
+@pytest.mark.parametrize(
+    "agg_func", [["min", "max"], "min", {"B": ["min", "max"], "C": "sum"}],
+)
+def test_multifunc_notimplimented(agg_func):
+    data = DataFrame(
+        {0: ["a", "a", "b", "b", "a"], 1: [1.0, 2.0, 3.0, 4.0, 5.0]}, columns=[0, 1]
```

Comment:
don't we allow user passed functions here?

Reply:
or are these tested above? e.g. do we have something that tests that we raise appropriately if the udf the user is passing doesn't accept the right args

Reply:
Yeah the tests above check that the udf should have the correct signature.